data/chains/hyper-evm.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "hyper-evm",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://hyperliquid.cloud.blockscout.com/api"
+    },
+    "browserUrl": "https://hyperliquid.cloud.blockscout.com/"
+  },
+  "id": "999",
+  "name": "HyperEVM",
```

Comment:
Similarly, HyperEVM identifies EVM front of Hyperliquid chain. So I'd name this chain as `Hyperliquid`.